lua/lvim/utils/functions.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+M.smart_quit = function()
+  if vim.api.nvim_eval [[&modified]] == 1 then
```

Comment:
```suggestion
  local bufnr = vim.api.nvim_get_current_buf()
  local modified = vim.api.nvim_buf_get_option(bufnr, "modified")
  if modified then
```

I think this should work the same, at least it seems fine in my testing.